### PR TITLE
Add more tests for PaintSweepGradient

### PR DESCRIPTION
**Pull request by drott**
Tests covering repeat modes, starting angle different from zero, start
angle and end angle difference larger than 360 degrees, negative angles,
counter-clockwise start angle and end angle sector crossing 0/360
degrees.

Intended to help fix Skia issues 13208, 13209, 13210

### CODE REVIEW

File: config/more_samples-glyf_colr_1.py [hunk 1]
```diff
@@ -46,40 +46,66 @@ def _cpal(color_str, alpha=1.0):
...
+    color_line_range_map = {
+        "narrow": {
+            "ColorStop": [
+                (0.25, *_cpal("red")),
+                (1 / 3 * 0.5 + 0.25, *_cpal("yellow")),
+                (2 / 3 * 0.5 + 0.25, *_cpal("green")),
+                (0.75, *_cpal("blue")),
+            ]
+        },
+        "wide": {
```
Comment (anthrotype) [inline config/more_samples-glyf_colr_1.py:71]: "wide" seems unused?

File: config/more_samples-glyf_colr_1.py [hunk 2]
```diff
@@ -1115,6 +1141,33 @@ def main():
...
         _colrv0_colored_circles(palette_test_colors, next(access_chars)),
         _colrv1_colored_circles(palette_test_colors, next(access_chars)),
+        # Sweep with repeat mode pad
+        _sample_sweep(0, 90, "pad", "narrow", next(access_chars)),
+        _sample_sweep(45, 90, "pad", "narrow", next(access_chars)),
+        _sample_sweep(247.5, 292.5, "pad", "narrow", next(access_chars)),
+        _sample_sweep(90, 270, "pad", "narrow", next(access_chars)),
+        _sample_sweep(-270, 270, "pad", "narrow", next(access_chars)),
+        _sample_sweep(-45, 45, "pad", "narrow", next(access_chars)),
+        _sample_sweep(315, 45, "pad", "narrow", next(access_chars)),
```
Comment (anthrotype) [inline config/more_samples-glyf_colr_1.py:1151]: should you also test a case where start angle > end angle (and does not wrap around 360), e.g. start=90, end=45? I'd espect the sweep to proceed clock-wise

Comment (anthrotype) [reply config/more_samples-glyf_colr_1.py:1151]: > I'd espect the sweep to proceed clock-wise

no, actually sweep always goes counter-clockwise by definition

https://docs.microsoft.com/en-us/typography/opentype/spec/colr#sweep-gradients

**Review by anthrotype — APPROVED** [review]

File: config/more_samples-glyf_colr_1.py (hunk 1 above)
Comment (drott) [reply config/more_samples-glyf_colr_1.py]: Removed the wide entry and added comment that this is preparation for testing with different ColorLines.

**Pull request merged**